### ISSUE TITLE
Broaden the scope of the ID search in the admin pages

### ISSUE THREAD

**Issue by eddierubeiz** [sciencehistory/scihist_digicoll#2658]
On these pages:
- https://digital.sciencehistory.org/admin
- https://digital.sciencehistory.org/admin/collections
filtering on an ID should match on any values of `external_id`: bib record ID, accession number, etc.

**Comment by jrochkind** [comment]
@eddierubeiz Is this one actually done? We never put it in review and reported it, I don't think

**Comment by eddierubeiz** [comment]
Yes, this is done - you had a small suggestion which I implemented. Glad you noticed it!
